scripts/update_config.js

```diff
@@ -0,0 +1,131 @@
+/*
+ * LiskHQ/lisk-scripts/updateConfig.js
+ * Copyright (C) 2017 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+const fs = require('fs');
+const readline = require('readline');
+const program = require('commander');
+const extend = require('extend');
+const lisk = require('lisk-js');
+
+/* eslint-disable no-console */
+
+program
+	.version('0.1.1')
+	.option('-o, --old <path>', 'Old config.json')
```

Comment:
Why make this an option if it's not optional?

Reply:
@willclarktech I don't find any other way to provide a required option in `commander` I ported the script from other repo, so most of boilerplate is the same. Do you know is there any option to specify required option in commander? 
https://www.npmjs.com/package/commander

Reply:
Isn't that what the `.parse(process.argv)` bit is for?